Source/WebCore/PAL/pal/text/TextEncodingRegistry.cpp

```diff
@@ -213,7 +213,7 @@ static void buildQuirksSets() WTF_REQUIRES_LOCK(encodingRegistryLock)
     ASSERT(!japaneseEncodings);
     ASSERT(!nonBackslashEncodings);
 
-    japaneseEncodings = new UncheckedKeyHashSet<ASCIILiteral>;
+    japaneseEncodings = new HashSet<ASCIILiteral>;
```

Comment:
I'm always annoyed by this idiom because `HashSet` is already a single pointer that starts out as a null so putting that pointer in a heap block is silly. Might be a nice tiny thing to clean up some day.

Reply:
Good point, I will follow-up shortly to take care of this.

Reply:
https://github.com/WebKit/WebKit/pull/47345